<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregar_propina_cuenta</class>
 <widget class="QDialog" name="agregar_propina_cuenta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="titulo">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(133, 133, 133);</string>
         </property>
         <property name="text">
          <string>Elija un porcentaje de propina (del total de la cuenta principal)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="diezPorciento">
         <property name="text">
          <string>10%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="quincePorciento">
         <property name="text">
          <string>15%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="veintePorciento">
         <property name="text">
          <string>20%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="otroMonto">
         <property name="text">
          <string>Monto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sinPropina">
         <property name="text">
          <string>Sin propina</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="OtroSelected" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="subtitulo">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(133, 133, 133);</string>
          </property>
          <property name="text">
           <string>Ingrese el monto de propina (en pesos):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="montoPropina">
          <property name="styleSheet">
           <string notr="true">border: none;
border-bottom: 3px solid #539E56;</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhHiddenText</set>
          </property>
          <property name="inputMask">
           <string>9999</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnSecundaio_2">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPrimario_2">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pagar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
